docs/PowerAuth-Enrollment-Server-1.5.0.md

```diff
@@ -9,24 +9,3 @@ This guide contains instructions for migration from PowerAuth Enrollment Server
 ### Drop MySQL Support
 
 Since version `1.5.0`, MySQL database is not supported anymore.
-
-
-### FDS Data
```

Comment:
Moved to the Onboarding migration guide. I believe that we want to keep it separate for those, who do have only the Enrollment server without Onboarding.